crm-contact-exporter.py

```diff
@@ -4,79 +4,92 @@
+
     data = response.json()
+
     access_token = data['access_token']
 
-    url = "https://www.zohoapis.eu/crm/v2/Contacts"
+    return access_token
+
+
+def get_contacts_from_zoho():
+    access_token = get_zoho_access_token()
 
-    payload = {}
-    files = []
     headers = {
         'Authorization': f'Zoho-oauthtoken {access_token}'
     }
 
-    response = requests.request("GET", url, headers=headers, data=payload, files=files)
+    response = requests.request("GET", os.environ.get("ZOHO_CONTACTS_URL"), headers=headers, data={}, files=[])
 
     zoho_list = response.json()
+
     data = zoho_list['data']
+
     contacts_list = []
+
     for contact in data:
-        contacts = {'email': contact['Email'], 'firstname': contact['First_Name'], 'lastname': contact['Last_Name'],
-                    'phone': contact['Phone'], 'title': contact['Title'], 'department': contact['Department']}
+        contacts = {'Department': contact['Department'], 'First name': contact['First_Name'],
+                    'Last name': contact['Last_Name'], 'Email': contact['Email'],
+                    'Phone': contact['Phone'], 'Title': contact['Title']}
         contacts_list.append(contacts)
-    contacts_zoho = pd.DataFrame(contacts_list).to_csv(f"reports/contacts_zoho-{today}.csv", index=False)
 
-    return contacts_zoho
+    zoho_contacts_df = pd.DataFrame(contacts_list)
+    zoho_contacts_df.to_csv(f"reports/zoho_contacts-{today}.csv", index=False)
+
+    return zoho_contacts_df
 
 
 def get_contacts_from_hubspot():
     api_client = HubSpot(
-        access_token="pat-eu1-8b6c2f2f-d75e-46b1-8a00-338ec3f66062")
-
-    api_client.access_token = "pat-eu1-8b6c2f2f-d75e-46b1-8a00-338ec3f66062"
+        access_token=os.environ.get("HUBSPOT_ACCESS_TOKEN"))
 
     hubspot_list = api_client.crm.contacts.get_all()
-    # print(hubspot_list)
 
     contacts_list = []
     for contact in hubspot_list:
-        contacts = {'email': contact.properties['email'], 'firstname': contact.properties['firstname'],
-                    'lastname': contact.properties['lastname']}
+        contacts = {'First name': contact.properties['firstname'], 'Last name': contact.properties['lastname'],
+                    'Email': contact.properties['email']
+                    }
         contacts_list.append(contacts)
-    contacts_hubspot = pd.DataFrame(contacts_list).to_csv(f"reports/contacts_hubspot-{today}.csv", index=False)
-    return contacts_hubspot
+
+    hubspot_contacts_df = pd.DataFrame(contacts_list)
+    hubspot_contacts_df.to_csv(f"reports/contacts_hubspot-{today}.csv", index=False)
+
+    return hubspot_contacts_df
 
 
 def get_contacts_from_salesforce():
-    sf = Salesforce(username=username,
-                    password=password,
-                    security_token=token,
-                    instance_url=instance)
+    sf = Salesforce(username=os.environ.get("SALESFORCE_USERNAME"),
```

Comment:
Remember a rule: no abbreviations in naming. What will you think if you will see 'rt' variable in the code you need to review? It is a rude naming violation. Only general abbreviations in Python are allowed: df, dt, os